### PR TITLE
Healthz updates

### PR DESCRIPTION
**Pull request by bartr**
Partially addresses #90

- Added ExactMatch Test
- Updated /healthz to use exact match

### CODE REVIEW

File: src/Smoker/SmokeTest.cs [hunk 1]
```diff
@@ -92,9 +92,11 @@ public async Task<bool> Run()
                         // validate the response
                         if (resp.StatusCode == System.Net.HttpStatusCode.OK && r.Validation != null)
                         {
+                            res = ValidateStatusCode(r, resp);
                             res = ValidateContentType(r, resp);
                             res += ValidateContentLength(r, resp);
```
Comment (aflinchb) [inline src/Smoker/SmokeTest.cs]: Should the 2nd line be: res *+=* ValidateContentType.. ? (Same for other validate blocks below)

File: src/Smoker/SmokeTest.cs [hunk 2]
```diff
@@ -150,10 +152,12 @@ public async Task<string> RunFromWebRequest()
                         // validate the response
                         if (resp.StatusCode == System.Net.HttpStatusCode.OK && r.Validation != null)
                         {
-                            res += ValidateContentType(r, resp);
+                            res = ValidateStatusCode(r, resp);
+                            res = ValidateContentType(r, resp);
                             res += ValidateContentLength(r, resp);
                             res += ValidateContains(r, body);
                             res += ValidateJsonArray(r, body);
```
Comment (aflinchb) [inline src/Smoker/SmokeTest.cs]: Add  res += ValidateExactMatch(r, body); here ?

File: src/Smoker/SmokeTest.cs [hunk 3]
```diff
@@ -243,6 +247,7 @@ public async Task RunLoop(int id, Helium.Config config, CancellationToken ct)
                                 res += ValidateContentLength(r, resp);
                                 res += ValidateContains(r, body);
                                 res += ValidateJsonArray(r, body);
+                                res += ValidateJsonObject(r, body);
                             }
```
Comment (aflinchb) [inline src/Smoker/SmokeTest.cs]: And add the exact match validation here too?

File: src/Smoker/SmokeTest.cs (hunk 1 above)
Comment (bartr) [reply src/Smoker/SmokeTest.cs]: I added a ValidateAll() to ensure consistency across run once, run from web and run loop

File: src/Smoker/SmokeTest.cs (hunk 2 above)
Comment (bartr) [reply src/Smoker/SmokeTest.cs]: added ValidateAll() to resolve

**Pull request merged**